docs/releases_and_versioning.rst

```diff
@@ -11,9 +11,9 @@ This documentation release, |release|, applies to the following versions of each
 ===================         ==============
 Component                   Version
 ===================         ==============
-|cf-long|                   v1.0.x
-|kctlr-long|                v1.0.x-1.3.x
-|mctlr-long|                v1.0.x, v1.1.x
+|cf-long|                   v1.0.x-1.1.x
+|kctlr-long|                v1.0.x-1.4.x
+|mctlr-long|                v1.0.x-v1.1.x
```

Comment:
We will be releasing marathon-bigip-ctlr v1.2.0 as part of the new set of releases this week.